article/app/views/liveblog/liveBlogBlocks.scala.html

```diff
@@ -42,4 +42,7 @@ <h2 class="block-title" itemprop="headline">@title</h2>
         }
         @views.html.fragments.share.blockLevelSharing(s"block-${block.id}", article.sharelinks.elementShares(s"block-${block.id}", None), article.metadata.contentType, amp)
     </div>
+    @block.attributes.membershipPlaceholder.map { membershipPlaceholder =>
```

Comment:
Yes. Right now it will never have a value because there's no way to supply it in Composer (see https://github.com/guardian/flexible-content/pull/2809). But we included it in the CAPI representation in case we ever need it in future